File: sum_of_a_series.swift
func invsqr(n:Int) -> Float {
  return 1.0 / (Float(n) * Float(n))
}

func sumSeriesTo(n:Int) -> Float {
  var counter: Float = 0
  for var i = 1; i <= 1000; i++ {
    counter += invsqr(i)
  }
  return counter
}

println(sumSeriesTo(1000))

File: sum_of_a_series.rb
# http://rosettacode.org/wiki/Sum_of_a_series

def invsqr(n)
  1.0 / (n * n).to_r
end

def sum(n)
  counter = 0.0
  (1..n).each do |num|
    counter += invsqr(num)
  end
  counter
end

p sum(1000)
